### PR TITLE
Member update

### PR DESCRIPTION
**Pull request by hmcty**
Updated members list, added past members section, added my photo

### CODE REVIEW

File: _data/members.yml [hunk 1]
```diff
@@ -1,86 +1,107 @@
...
+    year: 2023
+
+  - name: Adrian Teigen
+    image: 
+    major: Computer Science
+    year: 2021
+
+past:
+  - name: Victor Pan
+    position: Vice President
```
Comment (jrbaxter0) [inline _data/members.yml]: VP is no longer VP

File: _pages/members.md [hunk 2]
```diff
@@ -3,12 +3,11 @@ layout: archive
 permalink: /members/
 title: Members
 ---
-{% for member in site.data.members %}
+{% for member in site.data.members.current %}
   {% include member-grid.html %}
 {% endfor %}
 
-<!-- <div>
+# Past Members
```
Comment (jrbaxter0) [inline _pages/members.md]: I never got this to work right. I see a gap on my end. Do you too?

**Review by jrbaxter0 — CHANGES_REQUESTED** [review]
I think normally .png files are preferred, but I don't care that much. I would like to fix the VP issue though and at least briefly look into this gap thing.

File: _pages/members.md (hunk 2 above)
Comment (hmcty) [reply _pages/members.md]: It's an issue directly related to the "past members" header. I tried debugging for a little bit but to no avail. We can dig deeper, create a separate past-members page, or say good enough. Thoughts?

Comment (jrbaxter0) [reply _pages/members.md]: That's what I found too. Let's make an Issue just to track this, and only display current members for now. You can comment or delete the past members stuff in the page, but leave the yaml file as is

**Comment by hmcty** [comment]
Left out previous members but fixed VP issue. I'll go ahead and create issues for the gap and image formatting.

**Review by jrbaxter0 — APPROVED** [review]

**Pull request merged**